FriendlyMenu/ViewModels/DishSummaryVM.cs

```diff
@@ -6,10 +6,21 @@ namespace ViewModels
 {
     public class DishSummaryVM
     {
-        public string Name { get; set; }
```

Comment:
Do we need this?

Reply:
we have an DishName Property, so we delete it